tools/src/bin/docgen/main.rs

```diff
@@ -48,57 +45,6 @@ struct ApiDocs {
     aliases: Vec<TypeAliasItem>,
 }
 
-impl ApiDocs {
```

Comment:
I don't see why it makes sense to do these in the same commit? It certainly makes reviewing harder...

Reply:
Separated the cross-linking improvements into their own commit.